### ISSUE TITLE
Nuget package for last version before upgrading to .net core 3

### ISSUE THREAD

**Issue by TFTomSun** [BlazorComponents/MatBlazor#31]
Hi,

I just started to use your project and i love it! Very clean and easy to use. I discovered that the version 0.4.5 doesn't have many of the controls you've already implemented. I would like to stick with .NET Core 2.x. Would you be so kind and generate a nuget package with latest state before you upgraded to .NET Core 3.x? Thank you.

**Comment by SamProf** [comment]
Thank you.

Version 0.5 already in Nuget, but still not components works.

**Comment by SamProf** [comment]
Now, I have made my upgraded release MatBlazor 0.6. (Upgraded to Blazor 0.9) and fix some problems.
I have started to test with Razor Components Application.

So, now MatBlazor works with client side Blazor 0.9.

I found that Razor Component Application not loaded empbedded resource content from my library - so, I need few days to fix this.
Also I have started new sprint to develop more components and imprrove existsing.
Thank you.
